docs/contributors/components.md

```diff
@@ -0,0 +1,43 @@
+# Storybook & Components
+
+This repo includes [Storybook](https://storybook.js.org) tooling so we can test and develop components in isolation.
+
+The storybook is automatically built and published to [GitHub pages](https://woocommerce.github.io/woocommerce-gutenberg-products-block/) on every push to the main branch. See [travis.yml](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/main/.travis.yml) for details.
+
+https://woocommerce.github.io/woocommerce-gutenberg-products-block/
+
+## Where are our components?
+We have components in a few folders, for different contexts.
+
+- [`assets/js/base/components`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/tree/main/assets/js/base/components)
+- [`assets/js/components`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/tree/main/assets/js/components)
+- [`assets/js/icons`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/tree/main/assets/js/icons)
+
+__`assets/js/base/components`__ are used in front-end code, as well as editor & admin. 
+These components help us build consistent interfaces across the front end (shopper) experience and elsewhere.
+Because they can be used in the front end and editor, components in this folder should:
+
+-  Perform efficiently - i.e. not adversely affect page performance/experience.
+-  Have lean dependencies - i.e. not bloat the payload unnecessarily.
+-  Look consistent in common themes; ideally should allow themes to adjust appearance as necessary.
+
+__`assets/js/components`__ are used in the editor UI for our blocks. 
+These components are typically more specialised to WooCommerce Blocks. 
```

Comment:
Given the comments, I'd remove this line. I would read this and despite the usage of "typically", I'd still discard this folder containing any components that can be re-used in other projects (which is incorrect). I think for this iteration of the docs let's stick to what is super clear and that's the fact that this specific folder is for editor only components.

Reply:
Works for me!

Reply:
96a97da